components/layout/section/section.module.css

```diff
@@ -16,6 +16,10 @@
   flex: 1;
 }
 
+.chiefScientistBio p {
```

Comment:
I can see how the `margin: 0` on the paragraph element makes sense for lining up the top of the bio section with the image, but we would lose the spacing in between the paragraphs. This mostly affect the longer bios with multiple paragraphs. The two solutions I see that I think could both work are to change the `margin:0` to a `margin-top:0`, and only have it apply to the first child. The other is to add a margin or padding to the image so that both sides of the grid line up. @mbwatson would have more insight on this, but I think it might be personal preference! 

Also, this `layout/section` folder is a different component from the strapi-generated sections defined in the `sections` folder. Let's discuss in our meeting today!

Reply:
good thinking. the zero margin does look better in this instance. i would suggest removing styling here completely, though, as styles for elemts in the markdown will be addressed when we define the mapping from DOM elements to React components within our Markdown component. what do you two think about that?

Reply:
It's going to be hard to get the text to always align without knowledge of the markdown styles. For example, the `p` margin is `1rem` while `h1` is `1.25rem`. We can hardcode in the markup style a `marginTop='-1rem'` which works as long as the scientists' bios always start with a `p`. The best option I think is like @suejinkim20 mentioned, using the child pseudoclass:

```css
.chiefScientistContainer > *:first-child {
  margin-top: 0;
}
```

which knocks out the top margin of the first child in the container, regardless of what markdown is present.

But I do agree we need to decide where this css rule goes, because it's currently sharing a stylesheet with the other component.

Reply:
@Woozl, I agree with you that it'll be hard to do this without having the markdown component to map to. As for the location of the css, if you merge in the `update/new-strapi-section-folder` branch into this branch, you'll get a dedicated css file in that folder called `components/strapi-sections/strapi-components/strapi-components.module.css`. We can always delete this file in the future if we end up defining these styles in other components.

@mbwatson , I agree that eventually, we will want a more unified, higher level style, but for now, it makes sense to me to leave the style as it is and refactor it when a new component is created (for example, an `ImageTextSection` to complement the existing `Section` component)